docs/errorhandling.rst

```diff
@@ -78,7 +78,7 @@ This can be done by registering error handlers.
 
 An error handler is a normal view function that return a response, but instead
```

Comment:
While you're at it - can you correct the grammar? It should be "that return`s` a response "

Reply:
OK. I'll do that. Thanks!